docs/src/pages/en/media-live-button.md

```diff
@@ -0,0 +1,115 @@
+
+```html
+<media-live-button media-time-is-live></media-live-button>
+```
+
+<h3>Alternate text</h3>
+
+<media-live-button media-time-is-live>
+  <span slot="text">Hello!</span>
+</media-live-button>
+
+```html
+<media-live-button>
+  <span slot="text">Hello</span>
+</media-pip-button>
+```
+
+<h3>Alternate indicator SVG</h3>
+
+<media-live-button media-time-is-live>
+  <svg slot="indicator" viewBox="0 0 16 16"><circle cx="8" cy="8" r="8"></circle></svg>
+</media-live-button>
+
+```html
+<media-live-button media-time-is-live>
+  <svg slot="indicator" viewBox="0 0 16 16"><circle cx="8" cy="8" r="8"></circle></svg>
+</media-live-button>
+```
+
+<h3>Alternate indicator text or font icon</h3>
+
+<media-live-button media-time-is-live>
+  <span slot="indicator">Hello!</span>
+</media-live-button>
+
+```html
+<media-live-button media-time-is-live>
+  <span slot="indicator">Hello!</span>
+</media-live-button>
+```
+
+<h3>Common Use: Text is also the indicator</h3>
+
+<style>
+  #textlive {
+    --media-live-indicator-off-icon-color: white;
+  }
+</style>
+<media-live-button id="textlive">
+  <span slot="indicator">LIVE</span>
+  <span slot="spacer"></span>
+  <span slot="text"></span>
+</media-live-button>
+<media-live-button id="textlive" media-time-is-live>
+  <span slot="indicator">LIVE</span>
+  <span slot="spacer"></span>
+  <span slot="text"></span>
+</media-live-button>
+
+```html
+<style>
+  #textlive {
+    --media-live-indicator-off-icon-color: white;
+  }
+</style>
+<media-live-button id="textlive">
+  <span slot="indicator">LIVE</span>
+  <span slot="spacer"></span>
+  <span slot="text"></span>
+</media-live-button>
+```
+
+## Attributes
+
+| Name            | Type      | Default Value | Description |
+| --------------- | --------- | ------------- | ----------- |
+| `media-time-is-live` | `boolean` | `false`| Include when the media time is at or close to the most current time or _live edge_. Clicking the button will not seek to live when true. [_Set automatically by media-controller._] |
+| `media-paused` | `boolean` | `false`| Include when the media is paused. The button will unpause the video if paused when seeking to live. [_Set automatically by media-controller._] |
+
+## Slots
+
+| Name    | Default Type | Description |
+| ------- | ------------ | ----------- |
+| `text` | `text` | The text content of the button, with a default of "LIVE". |
+| `indicator`  | `svg` | The default is an SVG of a circle that changes to red when the video or audio is live. Can be replaced wiht your own SVG or font icon. |
+| `spacer`  | `text` | A simple text space (`&nbsp;`) between the indicator and the text. |
+
+## CSS Vars
+
+See our [styling docs](./styling#Buttons)
+
+| Var    | Description |
+| ------ | ----------- |
+| `--media-live-indicator-icon-color` | The color of the indicator icon when the audio or video is live  |
+| `--media-live-indicator-off-icon-color` | The color of the indicator icon when the audio or video is *not* live  |
```

Comment:
thought:
- `--media-live-indicator-icon-color` for the default/non-live color
- `--media-live-indicator-live-icon-color` for the live color

Reply:
I see how you get there, and feel the awkwardness in it. Feels a little weird to have "live" in there twice, and I expect "live indicator color" to refer to the red not the "off" state.

I changed it to:
* `--media-live-indicator-color` - squarely focused on the color of live and re-useable
* `--media-live-button-icon-color` - because what it's really doing is overriding the default `--media-icon-color`

Make sense?

Reply:
This might be a good thing for our next media chrome weekly meeting agenda. A thought I'm having is:
Should this be two separate variables at all? Should we instead just assume folks can currently use attribute selectors and potentially eventually use [Custom States](https://developer.mozilla.org/en-US/docs/Web/API/CustomStateSet) (which we keep running into candidate use cases for)?

Reply:
You can't use attribute selectors to target the indicator color specifically of the built-in indicator. You'd have to override the default slot to use attribute selectors. Or we'd have to expose it as a part. I could be fine with the latter, but I'm not sure how that works if the part is also a slot, and then implications of that for skins/themes.

CustomStateSet is the dream. Do we have a sense for how long we're going to be waiting for those?